Dockerfile

```diff
@@ -60,6 +60,25 @@ RUN . $TB3_OVERLAY_WS/install/setup.sh && \
     ros2 security generate_artifacts -k keystore \
       -p policies/tb3_gazebo_policy.xml
 
+# install tools for footprinting and pentesting
```

Comment:
Sorry, just came to my mind, but could you move this chunk up before line 55, so that if we or folks need to regenerate permissions with a changed policy, they wouldn't have to break so much of the build cache that would otherwise apt install this again.

Reply:
Sure. Done at https://github.com/ros-swg/turtlebot3_demo/pull/19/commits/82b2cd6fc05c49f14d65a9d26839b0357706c6f8. I tested things afterwards and everything seems to be operating fine. A double check would be appreciated though. Ping @mikaelarguedas, @thomas-moulard.

Reply:
I'm tuning out for the day due to poor internet, I can give it a shot tomorrow morning if it hasnt been validated by then